LSP.sublime-settings

```diff
@@ -97,6 +97,12 @@
   // using a keyboard shortcut or the context menu.
   "show_code_actions": "annotation",
 
+  // Show code lens:
+  // "annotation" - show an annotation on the right when code actions are available
+  // "phantom" - show a phantom on the top when code actions are available
+  // "" - don't show code lens
```

Comment:
Contrary to code actions, code lenses are enabled only in one place. So they can be disabled already with

```js
"disabled_capabilities": {
    "codeLensProvider": true
}
```

The different settings for code actions are needed because they appear in the hover popup as well as in the view, as well as can be requested manually with a keybinding.

Reply:
Yeah that works too. I wrote the setting in a similar fashion since some people might prefer a quick toggle and I was already there. I'll remove this setting though if it's duplicating another.